src/components/footer.tsx

```diff
@@ -14,7 +14,11 @@ const Footer = () => {
           <Link href="/mentions-legales" legacyBehavior>
             <a>Mentions légales</a>
           </Link>
+          <a href="https://github.com/ChtiJS/" title="Voir le compte GitHub de ChtiJS"></a>
+          <a href="http://twitter.com/ChtiJS"><img src="/images/icons/uE001-twitter.svg" alt="ChtiJS sur Twitter" /></a>
```

Comment:
You should use CSS instead since this is only design icons not illustrations : https://github.com/nfroidure/blog/blob/master/assets/less/components/_social.less#L1-L32